docs/default-repository-settings.md

```diff
@@ -0,0 +1,85 @@
+# Default repository settings
+
+## General
+
+Everything not mentioned is left at the GitHub default.
+
+* Merging pull requests
+  * Allow merge commits: `unchecked`
+  * Allow squash merging: :heavy_check_mark:
+  * Allow rebase merging: `unchecked`
+  * (only allowing squash merging to ensure a clean history)
+* Automatically delete head branches: :heavy_check_mark:
+  * (so that merged branches are not left lying around)
+
+## Collaborators and Teams
+
+Documents [Community Membership](../community-membership.md) and
+[CONTRIBUTING](../CONTRIBUTING.md) define how permissions are typically set up
+for the repository.
+
+1. Every repository has three teams associated with it. Typically for the
+   repository `opentelemetry-foo` they will be named `foo-triagers`, `foo-approvers`,
+   and `foo-maintainers`. `foo-maintainers` is a child of `foo-approvers`, and
+   `foo-approvers` is a child of `foo-triagers`, as each child team always contains
+   a subset of people and defines a larger scope of privileges.
+2. Every member of `foo-maintainers` should be included in
+   `foo-approvers` and `foo-triagers` explicitly, with the "Maintainer" GitHub
+   privileges. This allows repository maintainers to invite new approvers and
+   triagers to the team.
+3. The team `foo-triagers` has `Triage` permissions for the repository. If repository
+   is using the Project Boards, `foo-triagers` should have `Write` permissions to
+   have access to the Project Boards. Do not add members of `foo-triagers` to
+   the CODEOWNERS file, as their approvals should not count towards the required
+   number of approvals for merging.
+4. The team `foo-approvers` has `Write` permissions for the repository.
+5. The team `foo-maintainers` has `Maintain` permissions for the repository.
```

Comment:
this section was moved over from `how-to-configure-new-repository.md`

I removed the image because it was outdated and didn't have anything that wasn't in the text